### PR TITLE
jwplayer script.onerror

### PR DESCRIPTION
**Pull request by chrisdugne**
catching error on loading script for jwplayer

**Testing Strategy**

- [ ] Already covered by tests
- [x] Manual testing
- [x] Unit testing

### CODE REVIEW

File: packages/@coorpacademy-components/src/molecule/video-player/jwplayer.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ class JWPlayer extends React.Component {
   }
 
   componentDidMount() {
+    const jwPlayerScript = document.getElementById('jw-player-script');
+    jwPlayerScript.onerror = function() {
+      this.props.onScriptError && this.props.onScriptError();
+    };
```
Comment (VincentCATILLON) [inline packages/@coorpacademy-components/src/molecule/video-player/jwplayer.js]: don't know what to think about it, but refer to document seems not to be a clean solution :/

Comment (chrisdugne) [reply packages/@coorpacademy-components/src/molecule/video-player/jwplayer.js]: that's where the script is attached 
https://github.com/micnews/react-jw-player/blob/master/src/react-jw-player.jsx#L53

BTW this is waiting for my PR on `react-jw-player` appending `onerror` properly
here: https://github.com/micnews/react-jw-player/blob/master/src/helpers/install-player-script.js#L5 without a fork

Comment (Fausto95) [reply packages/@coorpacademy-components/src/molecule/video-player/jwplayer.js]: We could prolly fork the `react-jw-player` repo and apply our changes while the PR in the original repo is not merged.
What do y'all think?

**Comment by VincentCATILLON** [comment]
Do you have any before / after UI changes ?

**Comment by chrisdugne** [comment]
well basically we had a black squeare before, and now I added a text.
for example on the slide player:

Screen Shot 2019-09-23 at 3 03 54 PM

File: packages/@coorpacademy-components/src/molecule/video-player/test/fixtures/jwplayer.js [hunk 2]
```diff
@@ -14,6 +14,9 @@ export default {
         height: '100%'
       }
     },
+    scriptErrorMessage:
+      'Could not load the video player. Get sure to whitelist this platform within your adblocker if you use one.',
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/molecule/video-player/test/fixtures/jwplayer.js]: Make sure

Comment (Fausto95) [reply packages/@coorpacademy-components/src/molecule/video-player/test/fixtures/jwplayer.js]: `If you're using one`

Comment (chrisdugne) [reply packages/@coorpacademy-components/src/molecule/video-player/test/fixtures/jwplayer.js]: I've added the mooc wording: 'An error occurred while loading the video. Please make sure you have authorized this platform on your adblocker or deactivate your adblocker.',

File: packages/@coorpacademy-components/src/molecule/video-player/jwplayer.js (hunk 1 above)
Comment (chrisdugne) [reply packages/@coorpacademy-components/src/molecule/video-player/jwplayer.js]: with the latest modification I'm not sure the PR on `react-jw-player` is relevant.
And I rather add few lines on `molecule/video-player/jwplayer.js` than add a fork.

**Pull request merged**